### DISCUSSION TITLE
[@types/three] OrbitControls zoom0 key typo

### DISCUSSION THREAD

**Discussion by ArturNiko** [DefinitelyTyped/DefinitelyTyped#65276] [Issues with a @types package] [answered]
Please fix `zoom0` key, currently it's assigned as `zoomO`

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment, marked as answer]
Thanks for the issue report! Do you mind creating an issue at https://github.com/three-types/three-ts-types for it? It's just easier to keep track of there to make sure it gets fixed.

**Reply by ArturNiko** [reply]
Ok, I will do that.
